xbmc/filesystem/CurlFile.cpp

```diff
@@ -41,53 +42,28 @@ using namespace XCURL;
 
 #define FITS_INT(a) (((a) <= INT_MAX) && ((a) >= INT_MIN))
 
-curl_proxytype proxyType2CUrlProxyType[] = {
-  CURLPROXY_HTTP,
-  CURLPROXY_SOCKS4,
-  CURLPROXY_SOCKS4A,
-  CURLPROXY_SOCKS5,
-  CURLPROXY_SOCKS5_HOSTNAME,
-};
+CurlProxy ProxyType2CurlProxyType(CCurlFile::ProxyType proxy) noexcept
+{
+  if (!g_advancedSettings.CanLogComponent(LOGCURL))
```

Comment:
either the conditional statement for the if is missing or the below switch is indented incorrectly

Reply:
yeah I removed it in next commit, was a merge error